apps/frontend/src/app/backups-overview/backups/backups/backups.component.html

```diff
@@ -36,7 +35,12 @@ <h1 class="header">Backups Overview</h1>
     </clr-datagrid>
   </div>
   <div class="clr-col-sm-12 clr-col-md-6">
-    <div id="backupSizeChart" style="width: 100%; height: 300px"></div>
+    <div class="btn-group">
+      <button class="btn" [class.btn-primary]="selectedTimeRange === 'week'" (click)="setTimeRange('week')">Last Week</button>
+      <button class="btn" [class.btn-primary]="selectedTimeRange === 'month'" (click)="setTimeRange('month')">Last Month</button>
+      <button class="btn" [class.btn-primary]="selectedTimeRange === 'year'" (click)="setTimeRange('year')">Last Year</button>
+    </div>
```

Comment:
Unfortunately the buttons do not work as intended. For example on selecting Last Week it shows values from 18.10-21.10

Reply:
Unresolved it, because it's still not working. 
<img width="583" alt="image" src="https://github.com/user-attachments/assets/b1d1244d-aaf9-48aa-8117-bce154178805">